### DISCUSSION TITLE
How to change indent and arrow colors?

### DISCUSSION THREAD

**Discussion by alloc33** [nvim-tree/nvim-tree.lua#1926] [Q&A]
<img width="557" alt="Screenshot 2023-01-18 at 2 57 19 PM" src="https://user-images.githubusercontent.com/45129790/213105243-2178e450-0e1e-4177-bb69-f5768bdcdb34.png">

**Comment by gegoune** [comment]
Change `NvimTreeIndentMarker` hlgroup.
